src/store/store.ts

```diff
@@ -331,8 +332,12 @@ export const useStore = defineStore('store', {
       }
     },
 
-    /** The Business Legal Name. */
+    /** The Business Legal Name (or Alternate Name if this is a firm). */
     getBusinessLegalName (): string {
+      if (!GetFeatureFlag('enable-legal-name-fix')) {
```

Comment:
More or less similar to what was done in Filings UI.

Reply:
Yes. There are differences in how business info, and thus alternate names, are loaded. This is fine.